src/client/src/rt-system/ServiceStub.ts

```diff
@@ -39,14 +39,16 @@ export class ServiceStub {
   /**
    * wraps a RPC up as an observable stream
    */
-  requestResponse<TResponse, TPayload>(
-    remoteProcedure: string,
+  createRequestResponseOperation<TResponse, TPayload>(
+    service: string,
+    operationName: string,
     payload: TPayload,
   ): Observable<TResponse> {
     const dto: SubscriptionDTO<TPayload> = {
       payload,
       Username: this.userName,
     }
+    const remoteProcedure = service + '.' + operationName
```

Comment:
Here and below two different string concatenation is used. Can we make it consistent please?

I prefer `` const remoteProcedure = `${service}.${operationName}\` `` I think.